packages/server-boilerplate-middleware/README.md

```diff
@@ -164,7 +164,7 @@ Defaults to:
 ```js
 const corsOptions = {
   allowedHeaders: [],
-  allowedMethods: ['GET', 'POST', 'PUT', 'DELETE', 'OPTIONS'],
+  allowedMethods: ['GET', 'POST', 'PATCH', 'PUT', 'DELETE', 'OPTIONS'],
```

Comment:
can you update the allowed Methods in  `line 158` as well on this readme?

Reply:
👌